File: app/src/main/java/com/revotechs/calculator/HistoryItem.java
package com.revotechs.calculator;

import java.util.Date;

/**
 * Project Calculator
 * <p>
 * Created on 26.05.2017
 *
 * @author CriticalGnome
 */

class HistoryItem {

    private Date date;
    private String expression;
    private String result;

    HistoryItem(Date date, String expression, String result) {
        this.date = date;
        this.expression = expression;
        this.result = result;
    }

    public Date getDate() {
        return date;
    }

    public String getExpression() {
        return expression;
    }

    public String getResult() {
        return result;
    }

}

File: app/src/main/java/com/revotechs/calculator/entitiy/HistoryItem.kt
package com.revotechs.calculator.entitiy

import java.io.Serializable

/**
 * Project Calculator
 * Created on 30.05.2017
 * @author CriticalGnome
 */
data class HistoryItem(
        var id: Long?,
        var date: String,
        var expression: String,
        var result: String,
        var comment: String?,
        var locked: Boolean) : Serializable
